server/api/topics.js

```diff
@@ -31,11 +31,12 @@ router.get("/:topic_id", passport.authenticate("jwt", { session: false }),(req,
 
 
 router.post("/create", function(req, res) {
-	console.log("work-create");
-
 	const newTopicName = req.body.topic_name;
```

Comment:
Also, do you wanna learn a new way to grab a value from an Object. You can do  
```
const  { topic_name } = req.body;
```
More [info here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring)

Reply:
thank you